src/org/usfirst/frc4904/robot/humaninterface/drivers/DefaultDriver.java

```diff
@@ -1,5 +1,7 @@
 package org.usfirst.frc4904.robot.humaninterface.drivers;
 
+
+import org.usfirst.frc4904.robot.commands.AutoShifter;
```

Comment:
Nothing should be added to DefaultDriver - it should remain empty. (In fact we will probably remove it)